File: lesson4/python3/lesson4_10.py
#!/usr/bin/python3

numb = 2

while  numb < 100:
    count = 0
    k_numb = 2
    while k_numb * k_numb <= numb:
        if numb % k_numb == 0:
            count += 1
        k_numb += 1
    if count == 0:
        print("numb =", numb)
    numb += 1

File: lesson4/c_c++/lesson4_10.cpp
#include <iostream>

int main() {

    int32_t numb;

    for (numb = 2; numb < 500; numb++) {
        int32_t count, k_numb;
        count = 0;
        for ( k_numb = 2; k_numb < numb; k_numb++) {
            if ( numb % k_numb == 0) {
                count++;
            }
        }
        if ( count == 0) {
            std::cout << "numb = " << numb << std::endl;
        }
    }
    return 0;
}
